frontend/src/pages/Afterword/index.js

```diff
@@ -102,51 +120,49 @@ class Afterword extends React.Component<Props, State> {
   ];
 
   render() {
+    const { afterwordMessage, participantId } = this.props;
+    const { scores } = this.state;
+
     return (
       <Container>
+        <WideHeader title="Thanks for taking the exercise." />
         <ContentContainer>
-          <Title>Thanks for taking the exercise.</Title>
-          {this.props.afterwordMessage && (
-            <MarkdownContainer>
-              <ReactMarkdown source={this.props.afterwordMessage} />
-            </MarkdownContainer>
+          {afterwordMessage && (
+            <>
```

Comment:
We will need the `Markdown` like it was previously. The admins write the text using markdown in the admin side of phishtray. What `ReactMarkdown` does is convert that content to markdown so it looks good